machine_spraak_streaming/machine_spraak_streaming_stack.py

```diff
@@ -1,19 +1,82 @@
 from aws_cdk import (
     # Duration,
+    BundlingOptions,
     Stack,
+    CfnOutput, 
+    aws_lambda as lambda_,
+
+    aws_apigatewayv2_alpha as apiGateway,
+    aws_apigatewayv2_integrations_alpha as integrations
     # aws_sqs as sqs,
 )
 from constructs import Construct
+from os import path
+dirname = path.dirname(__file__)
+
 
 class MachineSpraakStreamingStack(Stack):
 
     def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
 
-        # The code that defines your stack goes here
 
-        # example resource
-        # queue = sqs.Queue(
-        #     self, "MachineSpraakStreamingQueue",
-        #     visibility_timeout=Duration.seconds(300),
-        # )
+        # for each API route, we define its own lambda function to handle
+        # the incoming data.
+        # This could also be done with one larger lambda function that has if/else flows
+        # depending on the route.
+        connect_handler = lambda_.Function(self, "connectApiFunction",
+            runtime=lambda_.Runtime.PYTHON_3_9,
```

Comment:
Just wondering, is there a specific reason for using Python 3.9 here and Python 3.7 below in the message api?

Reply:
Sharp eye ;)

Yes, I used python 3.9 b default but later noticed that the SciPy layer is only compatible with python 3.7. Most likely the smartest solution is to change the other lambdas to 3.7 as well for consistency.